.circleci/config.yml

```diff
@@ -19,10 +19,15 @@ jobs:
           # fallback to using the latest cache if no exact match is found
           - gnar-style-
 
+      # update bundler
+      - run:
+          name: Update bundler
+          command: gem uninstall bundler && gem install bundler
```

Comment:
Will we always need this step, or is this just to get the current images up to where they need to be?

Reply:
We'll need this step as long as the circle images ship with ruby installed with bundler 1. For example, the ruby 2.7 images for circle ship with bundler 2, but it's also using a newer version of debian as their base, so I don't know if it's more due to the debian version, the ruby version, or something else entirely.